integrations/weaviate/src/haystack_integrations/document_stores/weaviate/document_store.py

```diff
@@ -49,6 +63,16 @@ def __init__(
         Create a new instance of WeaviateDocumentStore and connects to the Weaviate instance.
 
         :param url: The URL to the weaviate instance, defaults to None.
+        :param collection_settings: The collection settings to use, defaults to None.
+            If None it will use a collection named `default` with the following properties:
+            - _original_id: text
+            - content: text
+            - dataframe: text
+            - blob_data: blob
+            - blob_mime_type: text
+            - score: number
+            See the official `Weaviate documentation<https://weaviate.io/developers/weaviate/manage-data/collections>`_
```

Comment:
I'm just curious about this format for links...
Is it Sphinx? Is it supported in ReadMe?
Should it be `Link text <link URL>`?

Reply:
This should be handle by `pydoc` when converting the docstrings to Markdown. 

In any case we need to standardise the docstrings sooner than later.